script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -59,12 +59,12 @@ She didn't deny any of it.
 -- Page 676, Offset 26087.
 -- あんな苦しそうな顔で逃げ出すとか、こいつらが正しいようじゃないか……！
 If she's the one running away with that anguished face,
- isn't that like admitting those people are right...!?
+ isn't that like admitting these people are right...!?
 }
 [sha:705dbb6b83f6963e92bfa0aea292a1c37675bc63]{
 -- Page 677, Offset 26088.
 -- 「テメエも短絡思考だな。
-"So you're also the type to fly off the handle, huh.
+"Seems you're a simpleton too.
```

Comment:
It's not wrong on that part, but I think I changed it because he's not saying "Shiki is also _'the type'_..." but rather that he's the same kind of 短絡思考 as Noel.